app/main.py

```diff
@@ -1,3 +1,19 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable
+from functools import wraps
+
+
+def cache(func: Callable) -> Callable:
+    cached_data = dict()
+
+    @wraps(func)
+    def wrapper(*args) -> int:
```

Comment:
are u sure, that func will return int ? what if decorated func will return float or smth else ?